sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -92,25 +103,34 @@ export interface KeyProperties extends ParsedKeyVaultEntityIdentifier {
  * @interface
  * An interface representing a deleted key
  */
-export interface DeletedKey extends Key {
-  /**
-   * @member {string} [recoveryId] The url of the recovery object, used to
-   * identify and recover the deleted key.
-   */
-  readonly recoveryId?: string;
+export interface DeletedKey {
   /**
-   * @member {Date} [scheduledPurgeDate] The time when the key is scheduled
-   * to be purged, in UTC
-   * **NOTE: This property will not be serialized. It can only be populated by
-   * the server.**
+   * @member {string} [value] The key value.
    */
-  readonly scheduledPurgeDate?: Date;
+  keyMaterial?: JsonWebKey;
```

Comment:
`key`

Reply:
This won't be done as part of this PR. The issue tracking this change is #5577